SocketRocket/SRWebSocket.h

```diff
@@ -45,11 +45,16 @@ extern NSString *const SRWebSocketErrorDomain;
 // It will be nil until after the handshake completes.
 @property (nonatomic, readonly, copy) NSString *protocol;
 
+// Should not be changed after opening websocket
+@property (nonatomic) BOOL allowSelfSignedCertificates;
```

Comment:
Total nit, this should be `allowsSelfSignedCertificates` (https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingIvarsAndTypes.html)